Library/ViewModels/PledgeCTAContainerViewViewModel.swift

```diff
@@ -205,21 +205,29 @@ private func pledgeCTA(project: Project, backing: Backing?) -> PledgeStateCTATyp
     return .prelaunch(saved: projectIsSaved, watchCount: project.watchesCount ?? 0)
   }
 
+  let isInPostCampaign = featurePostCampaignPledgeEnabled() && project.isInPostCampaignPledgingPhase
+
   guard let projectBacking = backing, project.personalization.isBacking == .some(true) else {
     if currentUserIsCreator(of: project) {
       return PledgeStateCTAType.viewYourRewards
     }
 
+    if isInPostCampaign {
+      return PledgeStateCTAType.pledge
+    }
+
     return project.state == .live ? PledgeStateCTAType.pledge : PledgeStateCTAType.viewRewards
   }
 
   // NB: Add error case back once correctly returned
-  switch (project.state, projectBacking.status) {
-  case (.live, _):
+  switch (project.state, projectBacking.status, isInPostCampaign) {
+  case (.live, _, _):
     return .manage
-  case (.successful, .errored):
+  case (.successful, _, true):
+    return .viewBacking
+  case (.successful, .errored, _):
```

Comment:
I could be wrong here, but I think a user could have an errored backing even if the project is in a post campaign state. If that's the case, we'd want to check the errored case first, in which case I don't think we actually need any custom post campaign logic at all?